homeassistant/components/connectedcars/config_flow.py

```diff
@@ -0,0 +1,143 @@
+"""Config flow for ConnectedCars.io integration."""
+import logging
+
+from connectedcars.client import ConnectedCarsClient
+from connectedcars.constants import QUERY_USER, QUERY_VEHICLE_VIN
+from connectedcars.exceptions import ConnectedCarsException
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+
+from .const import DEFAULT_NAMESPACE, DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("namespace", default=DEFAULT_NAMESPACE): str,
+        vol.Required("username"): str,
+        vol.Required("password"): str,
```

Comment:
Use the constants available?
```suggestion
        vol.Required(CONF_NAMESPACE, default=DEFAULT_NAMESPACE): str,
        vol.Required(CONF_USERNAME): str,
        vol.Required(CONF_PASSWORD): str,
```